gradle.properties

```diff
@@ -0,0 +1,7 @@
+/*
+* Copyright (c) 2012-2023 IriusRisk SL. All rights reserved
+*/
+
+version=1.2.0-SNAPSHOT
```

Comment:
wich version should we have here? Its 1.1.0 on master. Should it be 1.1.0 also as we are not changing code?

Reply:
Yes, we keep version 1.2.0-SNAPSHOT